dashboards-observability/public/components/explorer/log_explorer.tsx

```diff
@@ -140,6 +140,8 @@ export const LogExplorer = ({
     return newTabId;
   };
 
+  // console.log("curent selected tab from log explorer ",curSelectedTabId);
```

Comment:
Can you do a second pass and remove any leftover console log statements?

Reply:
Yeah okay, i'll remove them